### PR TITLE
Navigation & Modal Components

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch is for the Navigation and Modal Components

### CODE REVIEW

File: Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+.navbar-left i {
+    margin-right: 2rem;
+}
+
+/* .navbar-right i {
+    margin-left: 2rem;
+} */
+
```
Comment (romabulani) [inline Navigation/navigation.css]: Remove the comments if not required

File: master.css [hunk 2]
```diff
@@ -9,4 +9,6 @@
 @import url("Input/input.css");
 @import url("Lists/lists.css");
 @import url("Text/text.css");
-@import url("Toast/toast.css");
\ No newline at end of file
+@import url("Toast/toast.css");
+@import url("Modal/modal.css");
+@import url("Navigation/navigation.css");
```
Comment (romabulani) [inline master.css:14]: You should always give new line at the end of the file, so this red error wont come

**Review by romabulani — APPROVED** [review]
- Classnames are quite self explanatory
- Good to merge

**Review by ashishnnnnn — APPROVED** [review]
Code looks understandable and clean..

**Pull request merged**